presentation/common/src/test/hierarchy/Node.test.ts

```diff
@@ -28,7 +28,7 @@ describe("Node", () => {
 
   describe("toPartialJson", () => {

```

Comment:
method names in `describe` should probably also be updated

Reply:
Thanks, fixed in 081db4834e0b43c3d1ae577432e210e338ed4a44.